targets/UnrealMarketplacePlugin/examplesource/ExampleProject/BuildMac.sh.ejs

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+DoWork() {
+    echo ========== BUILDING iOS ==========
```

Comment:
The script name says Mac, but this comment says iOS. Should it be consistent?

Reply:
Good catch, I'll fix.